### PR TITLE
feat: add method to get localizations for ContentItem

### PR DESCRIPTION
**Pull request by PikinerLiliya**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Tests (`npm run test`) for the changes have been added (for bug fixes / features)
- [x] Docs (`npm run doc`) have been reviewed and added / updated if needed (for bug fixes / features)
- [x] PR title and git commit messages conform to the [conventionalcommits](https://www.conventionalcommits.org/en/v1.0.0/) specification

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

Management SDK is missing a method to get localizations for ContentItem.

## What is the new behaviour?

Added method to get localizations for ContentItem.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/lib/model/LocalizationJob.spec.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import test from 'ava';
+import { MockDynamicContent } from '../DynamicContent.mocks';
+
+test('get fby Root Content Item', async (t) => {
```
Comment (easen-amp) [inline src/lib/model/LocalizationJob.spec.ts:4]: type in test name

**Review by easen-amp — CHANGES_REQUESTED** [review]
Just a typo in the test name

**Review by srweeks — COMMENTED** [review]
PR title does not conform to conventional commits

File: src/lib/model/LocalizationJob.spec.ts (hunk 1 above)
Comment (PikinerLiliya) [reply src/lib/model/LocalizationJob.spec.ts]: Fixed

**Comment by PikinerLiliya** [comment]
> PR title does not conform to conventional commits

Fixed, thanks

**Review by srweeks — APPROVED** [review]

**Review by srweeks — APPROVED** [review]

**Review by srweeks — COMMENTED** [review]
Agreed with @mdinning-amp that typing for localizations (only returns a subset of a content item) will be done in a follow up release

**Pull request merged**